<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загруженность </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="l_airportName">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabW_statistics">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_monthlyStatistic">
      <attribute name="title">
       <string>Статистика за месяц</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="cb_months">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wid_monthlyStatistic" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_yearlyStatictic">
      <attribute name="title">
       <string>Статистика за год</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="wid_yearlyStatistic" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_close">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
